examples/unitTestExample/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="io.realm.examples.intro" >
```

Comment:
Ooops. Forgot to change that. I merely copied over the project and started editing. I'll fix this!